tools/gce/linux_performance_worker_init.sh

```diff
@@ -105,17 +115,17 @@ nvm install 4 && npm config set cache /tmp/npm-cache
 nvm install 5 && npm config set cache /tmp/npm-cache
 nvm alias default 4
 
-# C# dependencies (http://www.mono-project.com/docs/getting-started/install/linux/#debian-ubuntu-and-derivatives)
-
+# C# mono dependencies (http://www.mono-project.com/docs/getting-started/install/linux/#debian-ubuntu-and-derivatives)
 sudo apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 3FA7E0328081BFF6A14DA29AA6A19B38D3D831EF
```

Comment:
I intentionally kept mono for two reasons:
-- it is useful to have the ability to run benchmarks under mono without needing to recreate/update all performance workers. 
-- i think it's necessary for running the net45 target even when compiled by dotnet cli